bin/Unicodable.p6

```diff
@@ -96,7 +96,7 @@ method from-numerics($query) {
     $query ~~ m:ignoremark/^
         :i \s*
         [
-            [ | ‘u’ (.) <?{ $0[*-1].Str.uniname.match: /PLUS.*SIGN/ }>
+            [ | [｢\U｣ |‘u’ (.)] <?{ $0[*-1].Str.uniname.match: /PLUS.*SIGN/ }>
```

Comment:
I think the closing `]` is misplaced and should be at the end of the line, otherwise it expects something like `\U+2665` but the tests expect otherwise.